src/index.html

```diff
@@ -12,6 +12,33 @@
             <h1 class="header-title">todo app</h1>
         </header>
 
+        <section class="search-container">
+            <form class="search-form">
+                <input type="search" name="search" class="input-search" />
+            </form>
+            <button class="btn-search">Search</button>
```

Comment:
If you use button for form, it should be button with type submit, and button will locate inside form

Reply:
I fixed @phuocduong-agilityio. Please help me check. Thanks!